Update example project

veerChauhan opened the pull request:
will close #338 #337 #321 #304 #303

skywinder commented:
Hello, @veerChauhan . 
thank you for the update!
 I see so many diffs in your PR, would like to clarify the details, what is example changed and how to run the updated example. Can you add some details, how to run in in readme?


💸I'm willing to give a bounty for your help (since I receive an Ethereum grant to support this library).

veerChauhan commented:
Hi @skywinder,

I have added a helper function not into the library but outside the library for creating a wallet, getting wallet details such as private keys, mnemonics which will help people developing wallets with this library. also, I have added the Keystore method so that they can get the keystore any time they want to do the transaction.   

and for example project I have used Webkit and removed web view so it's an update on the example.
